addon/core/say-controller.ts

```diff
@@ -63,6 +63,19 @@ export default class SayController {
     return this.mainEditorView.domParser;
   }
 
+  get editable() {
+    return this.mainEditorView.editable;
+  }
+
+  toggleEditable() {
```

Comment:
I tend to prefer explicit `setEditable(editable: boolean)` interfaces, they're more flexible than toggles, and toggle behavior is trivial to implement in the consumer